Cargo.toml

```diff
@@ -99,3 +99,7 @@ flight = [
 ]
 duckdb-federation = ["duckdb"]
 sqlite-federation = ["sqlite"]
+postgres-federation = ["postgres"]
+
+[patch.crates-io]
+duckdb = { git = "https://github.com/spiceai/duckdb-rs.git", rev = "f2ca47d094a5636df8b9f3792b2f474a7b210dc1" }
```

Comment:
Same here

Reply:
I was not able to replace this with the crate version. 

We should keep this PR open, and upstream the changes from spiceai/duckdb-rs to duckdb. what do you think?

The issue is coming from this line: https://github.com/hozan23/datafusion-table-providers/blob/6e33170f28076457bbd643e4b9fc07ca2cd450ad/src/sql/db_connection_pool/dbconnection/duckdbconn.rs#L285-L287

Reply:
Yeah, we've had the PR open for a while now. https://github.com/duckdb/duckdb-rs/pull/373